config_spec.toml

```diff
@@ -0,0 +1,19 @@
+[[param]]
+name = "address"
+type = "String"
+optional = false
+doc = "LND GRPC address, kindly note that the address must start with https:// ."
```

Comment:
Unnecessary space in "// ." Perhaps also note that the format is `https:// ip address : port`

Reply:
Thank you.